tests/Yaapii.Atoms.Tests/Map/SortedTest.cs

```diff
@@ -104,8 +104,12 @@ public void DoesNotBuildValueWhenNotNeeded()
             Assert.Equal(1, keys[1]);
             Assert.Equal(6, keys[2]);
 
-            var ex = Assert.Throws<Exception>(() => sorted.Values.GetEnumerator());
-            Assert.Equal("i shall not be called", ex.Message);
+            // Sanety check: building one value isn't a problem
+            Assert.Equal(4, sorted[1]);
```

Comment:
we have the rule that we make only one assert per test. Please refactor.
Please remove comments (the method names of the tests, which are facts, will tell us what your comment currently does)